Network/DNS/Transport.hs

```diff
@@ -70,6 +75,7 @@ type UdpRslv = Int -- Retry
 resolve :: Domain -> TYPE -> Resolver -> Rslv0
 resolve dom typ rlv qctls rcv
   | isIllegal dom = return $ Left IllegalDomain
+  | typ == AXFR   = return $ Left NonZoneTransferAXFRRequest
```

Comment:
Bikeshedding: What do you think of `InvalidAXFRLookup` instead of the longer `NonZoneTransferAXFRRequest`? Matter of taste, the original is not confusing or anything, just a bit long...

Reply:
Makes sense to me. I made the change